app/views/shared/_comments.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if (post.comments.empty?) %>
+  <p>No comment yet.</p>
+<% else %>
+
+  <% post.recent_comments.each do |comment| %>
+    <ul class="border-black ">
+      <li><%= comment.author.name %>: <%= comment.text %></li>
+    </ul>
```

Comment:
- [ Optional ] It would be better if the `_comments.html.erb` partial only receives an object with `comments` to be rendered because you are using it to display `post index` which shows recent comments and `post show` page which is supposed to show all the comments. Currently, it renders the most recent comments only even if I've opened the posts show page. Please see the screenshot below for more details on this.

## Posts show page
![Screenshot from 2023-03-05 00-58-38](https://user-images.githubusercontent.com/16392046/222930522-e6631181-657a-4435-aa73-1b2e7bda4c9b.png)